src/style.css

```
@@ -1 +1,40 @@
-/* add styles here */
+.page__body {
+  margin: 0;
+}
+
+.stars-block {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
```

you can use  `:not(:last-child)` instead
```suggestion
  background-image: url(/src/images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}

.stars__star:not(:last-child) {
  margin-right: 4px;
}
```